lib/ansible/config/ansible_builtin_runtime.yml

```diff
@@ -6711,11 +6713,11 @@ plugin_routing:
     ovirt_vnic_profile_info:
       redirect: ovirt.ovirt.ovirt_vnic_profile_info
     dellos10_command:
-      redirect: dellemc.os10.os10_command
+      redirect: dellemc_networking.os10.os10_command
```

Comment:
Needs reverting

Reply:
Need to fix this in devel then in stable.2.10, so in another PR

Reply:
#70747